### ISSUE TITLE
Wind speed description of Fig. 7.8 in lecture08 notes is inconsistent with code graph

### ISSUE THREAD

**Issue by jaspermatthe** [awecourse/resources#4]
Fig. 7.8 of lecture08 notes has an x-axis range of 0 to 20 m/s whereas the supporting text below the figure on pages 22 and 23 state wind speeds of up to 41 m/s:
![image](https://github.com/awecourse/resources/assets/69607411/86035ead-e422-4e1e-bd99-61c84b43728d)
page 22:
![image](https://github.com/awecourse/resources/assets/69607411/05482cfe-d321-4188-aa83-e6cddd28c4d3)
page 23:
![image](https://github.com/awecourse/resources/assets/69607411/ae07dde7-8731-4dbb-a84d-ce424f685ce8)
I assume this may have been copied over from the "Chapter 6 Scaling Airborne Wind Energy Systems for Deployment on Mars" (https://doi.org/10.1007/978-3-031-50081-7_6) which these lecture08 notes were derived from. As hypothesized, the Fig. 7.14 from the paper on Mars has mentions of wind speeds of 41 m/s:
![image](https://github.com/awecourse/resources/assets/69607411/6086f023-6af3-4276-a1f8-c5320c7798c6)

**Comment by rschmehl** [comment]
Yes, the re-adaptation of the model to terrestrial applications is likely the reason for this bug. Thanks for pointing me to this (and providing a solution)!
